mqtt-streaming/src/main/scala/akka/stream/alpakka/mqtt/streaming/scaladsl/MqttSession.scala

```diff
@@ -226,6 +233,10 @@ final class ActorMqttClientSession(settings: MqttSessionSettings)(implicit mat:
           reply.future.map {
             case Unsubscriber.ForwardUnsubAck(carry: Option[A] @unchecked) => Right(Event(cp, carry))
           }
+        case Right(cp @ Publish(flags, _, Some(packetId), _)) if flags.contains(ControlPacketFlags.DUP) =>
```

Comment:
Any implications here for a broken/malicious client that e.g. sets the DUP flag when it isn't really one?

Reply:
In this instance, a publish will be dropped given that there's no packet id already registered.